src/Domain/FunctionsOfTime/PiecewisePolynomial.cpp

```diff
@@ -70,7 +70,10 @@ PiecewisePolynomial<MaxDeriv>::func_and_derivs(const double t) const noexcept {
 
 template <size_t MaxDeriv>
 void PiecewisePolynomial<MaxDeriv>::update(
-    const double time_of_update, const DataVector& updated_max_deriv,
+    // The NOLINT is for performance-unnecessary-value-param
+    // where clang-tidy says to use 'const DataVector& updated_max_deriv'.
+    // However, updated_max_deriv is std::moved out of inside this function.
+    const double time_of_update, DataVector updated_max_deriv,  // NOLINT
```

Comment:
Use `NOLINT(performance-unnecessary-value-param)` (or the `NOLINTNEXTLINE` version).